radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
```

Comment:
This function prepares the feature class to calculate features (i.e. this function calculates the texture matrix, upon which the features are calculated).
The reason for the name is so that there can be a second, parallel function for voxel-wise calculation, which can be called `_initVoxelWiseCalculation`.

I'll add a docstring, but because this is a private function, this will only be visible in the source code, not the documentation generated by sphinx.